database/seeds/eggs/source-engine/egg-garrys-mod.json

```diff
@@ -8,7 +8,7 @@
     "author": "support@pterodactyl.io",
     "description": "Garrys Mod, is a sandbox physics game created by Garry Newman, and developed by his company, Facepunch Studios.",
     "image": "quay.io\/pterodactyl\/core:source",
-    "startup": ".\/srcds_run -game garrysmod -console -port {{SERVER_PORT}} +ip 0.0.0.0 +map {{SRCDS_MAP}} +gamemode {{GAMEMODE}} -strictportbind -norestart +sv_setsteamaccount {{STEAM_ACC}} +host_workshop_collection {{WORKSHOP_ID}} +maxplayers {{MAX_PLAYERS}}  -tickrate {{TICKRATE}}",
```

Comment:
Why did you remove +maxplayers {{MAX_PLAYERS}} to add allocation limit to do the same thing?

Reply:
To have a consistent way of managing the maximum allowed players on a server, instead of using allocation_limit on a server, max_players on another, maximum_slots on another and so on.